### PR TITLE
Add favourites to move items type.

### PR DESCRIPTION
**Pull request by suchyindustries**

### CODE REVIEW

**Comment by cjshrader** [comment]
I'm not sure what the use case is for this, but at the same time I don't see how more options hurt anyone.

Can you put the enum in alphabetical order? I know @chrisoro likes to keep it that way to make them easier to read (so put favorites after everything).

Also could you update the readme with this new option?

File: src/config/models.py [hunk 1]
```diff
@@ -34,10 +34,10 @@ class HandleRaresType(enum.StrEnum):
 
 
 class MoveItemsType(enum.StrEnum):
+    favorites = enum.auto()
     everything = enum.auto()
```
Comment (cjshrader) [inline src/config/models.py:37]: Move favorites down one more, it should be after everything to keep it alphabetical. I'd do it myself but I don't think I can

**Review by cjshrader — CHANGES_REQUESTED** [review]

**Review by cjshrader — APPROVED** [review]

**Comment by cjshrader** [comment]
this looks good to me, but I don't have final approval of course.

**Comment by suchyindustries** [comment]
> I'm not sure what the use case is for this, but at the same time I don't see how more options hurt anyone.
> 
> Can you put the enum in alphabetical order? I know @chrisoro likes to keep it that way to make them easier to read (so put favorites after everything).
> 
> Also could you update the readme with this new option?

Personally, I missed favourites option from start of this feature. Move junk to inventory (to sell or salvage), and move favourites to stash (keep them for market or alt)

**Review by chrisoro — DISMISSED** [review]

**Review by chrisoro — APPROVED** [review]

**Pull request merged**